docs/pages/try-out-teleport/browser-labs.mdx

```diff
@@ -8,7 +8,7 @@ You can quickly try out some of Teleport's key features from your browser.
 
 Choose one of our interactive learning tracks, which are hosted by Instruqt:
 
-- [Try Teleport Server Access](https://goteleport.com/labs/teleport-demo/): Try Teleport Server Access, which makes it easier to configure onboarding, RBAC, and auditing for SSH connections to remote hosts.
-- [Try Teleport Application Access](https://goteleport.com/labs/teleport-app-demo/): Try Teleport Application Access, which gives you secure access to your internal web applications.
-- [Try Teleport Database Access](https://goteleport.com/labs/teleport-database-access-demo/): Learn how to use Teleport to securely access and configure a remote MySQL database.
-- [Try Teleport Kubernetes Access](https://goteleport.com/labs/teleport-kubernetes-demo/): Try Teleport Kubernetes Access, which provides advanced RBAC controls and auditing for `kubectl` commands.
+- [Try Teleport Server Access](https://play.instruqt.com/teleport/invite/imz8g1n1xzru): Try Teleport Server Access, which makes it easier to configure onboarding, RBAC, and auditing for SSH connections to remote hosts.
```

Comment:
Now that all of these links point to the same URL, should we condense them into a single link instead? E.g., "Choose one of our interactive learning tracks, which are hosted by Instruqt:" becomes:

```markdown
Try our [interactive learning tracks](https://play.instruqt.com/teleport/invite/imz8g1n1xzru), which are hosted by Instruqt.
```

Reply:
I think it's better to list all tracks, even with the same link so it's clear which tracks are available to try.

Reply:
You're both right. Having multiple copies of the same link can disorient the reader, but listing available labs is a good thing. How about something like this:

```
Choose one of our [interactive learning tracks](https://play.instruqt.com/teleport/invite/imz8g1n1xzru), which are hosted by Instruqt. These labs cover:

- Teleport Server Access, which makes it easier to configure onboarding, RBAC, and auditing for SSH connections to remote hosts.
- Teleport Application Access, which gives you secure access to your internal web applications.
- Teleport Database Access, to learn how to securely access and configure a remote MySQL database.
- Teleport Kubernetes Access, which provides advanced RBAC controls and auditing for `kubectl` commands.
```

Reply:
Thanks, @alexfornuto and @ptgott . I've taken your feedback onboard and updated the PR. PTAL.